docs/source/users/options/output_option.rst

```diff
@@ -1,10 +1,10 @@
-.. _output_option:
+.. _plotting_option:
 
-##############
-Output Options
-##############
+################
+Plotting Options
+################
 
-The output section contains options to control how results are outputted.
+The plotting section contains options to control how results are outputted and presented.
```

Comment:
I think we should merge `Plotting` into `Output`, not `Output` into `Plotting`; plotting is a form of output, but I'd never to think to look for where to set the output directory under a section called `Plotting`.